doc/source/getting_started/installing.rst

```diff
@@ -0,0 +1,458 @@
+
+.. code-block:: bash
+
+   $ pip install --user odl
+
+.. warning::
+    This will probably not work on Windows, since compiled dependencies (e.g. Numpy) will fail to build.
+    Detailed instuctions for these steps is beyond the scope of this document.
+
+
+Extra dependencies
+------------------
+The following optional packages extend the functionality of ODL.
+They can be specified as keywords in square brackets, separated by commas (no spaces!):
+
+.. code-block:: bash
+
+   $ pip install odl[dep1,dep2]
+
+Possible choices:
+
+- ``all`` : Install with all extra dependencies.
+- ``show`` : Install matplotlib_ to enable displaying capabilities.
+- ``fft`` : Install `pyFFTW`_ for fast Fourier transforms. Note that this requires the `FFTW`_ C library to be available on your system.
+  Note also that even without this dependency, FFTs can be computed with Numpy's FFT library.
+- ``pywavelets`` : Install `PyWavelets`_ for wavelet transforms.
+- ``scikit`` : Install `scikit-image`_ as a simple backend for ray transforms.
+- ``proximal``: Install the `ProxImaL`_ convex optimization package.
+
+
+These dependencies are optional and may not be easy to install on your system (especially on Windows).
+In general, a clean ODL installation is enough for most users' initial needs.
+
+
+------
+
+
+********************************************************************
+Alternative 3: Installation from source (recommended for developers)
+********************************************************************
+This installation method is intended for developers who want to make changes to the code.
+It assumes that the `Git`_ version control system is available on your system; for up-to-date instructions, consult the `Git installation instructions <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_.
+You also need `pip`_ to perform the installation.
+
+.. note::
+    You should consider performing all described steps in a `conda environment <http://conda.pydata.org/docs/using/envs.html>`_ -- it gives you the same encapsulation benefits as developer that you would enjoy also as a user (no conflicting packages, free to choose Python version, ...).
+    See the `Installing Anaconda`_ section for setup instructions.
+
+To get ODL, clone the repository with the command
+
+.. code-block:: bash
+
+   $ git clone https://github.com/odlgroup/odl
+
+No GitHub account is required for this step.
+
+
+In a conda environment
+======================
+You can choose to install dependencies first (optional ones in square brackets):
+
+.. code-block:: bash
+
+    $ conda install nomkl numpy scipy future [matplotlib]
+
+After that, enter the top-level directory of the cloned repository and run
+
+.. code-block:: bash
+
+   $ pip install --editable .
+
+Using only ``pip``
+==================
+Enter the top-level directory of the cloned repository and run
+
+.. code-block:: bash
+
+   $ pip install --user --editable .
```

Comment:
Ah, but further up we're in a `conda` environment where it makes no sense to use `--user` since the default install location is user-writable.